scripts/migrate_score_override.py

```diff
@@ -16,32 +16,40 @@ def main():
     from openassessment.assessment.models import Assessment, AssessmentPart, StaffWorkflow
     from openassessment.workflow.models import AssessmentWorkflow, AssessmentWorkflowStep
     from student.models import anonymous_id_for_user
-    from submissions.models import Score, ScoreAnnotation, Submission
+    from submissions.models import Score, ScoreSummary, ScoreAnnotation
 
     old_scores = Score.objects.filter(submission__isnull=True, reset=False).order_by('id')
     for score in old_scores:
         try:
             with transaction.atomic():
-                points_override = score.points_earned
-                points_possible = score.points_possible
+                # ScoreSummary is updated on Score saves but for this script we don't want that.
```

Comment:
@kluo Could you provide some more details on why we don't want the ScoreSummary updated.

Reply:
Since we are not actually making a new score, just updating an old override, we want to maintain ScoreSummary to keep its same latest Score.